packages/internals/src/get-generators/getGenerators.ts

```diff
@@ -124,6 +127,15 @@ export async function getGenerators(options: GetGeneratorOptions): Promise<Gener
     throw new Error(missingDatasource)
   }
 
+  if (binaryTargetsOverride !== undefined) {
+    for (const generator of config.generators) {
+      generator.binaryTargets = binaryTargetsOverride.map((target) => ({
```

Comment:
This makes the CLI print the warning like `Warning: Your current platform `darwin-arm64` is not included in your generator's `binaryTargets` configuration ["rhel-openssl-1.0.x"]` which must be skipped.

Reply:
(converting to draft for now because of this)

Reply:
Maybe check how that changes once https://github.com/prisma/prisma/pull/19330 is merged?